pydicom/tests/test_json.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from pydicom.data import get_testdata_files
+from pydicom.dataelem import DataElement
+from pydicom.dataset import Dataset
+from pydicom import compat
+
+from pydicom.valuerep import PersonNameUnicode, PersonName3
+
+def test_json_PN():
+    s = open(get_testdata_files("test_PN.json")[0], "r").read()
```

Comment:
where is that file?

Reply:
Missed staging it, pushed up now.  I had at one point removed that again.  Was back and forth trying to figure things out.

Reply:
Ah, sorry, checked too early... as I wrote, I started to review this yesterday and checked that PatientName problem, but didn't have the time to finish... Anyway, I also agree that this can be merged as is after your fix. I haven't seen anything in the release notes yet, are you going to add it? (I may have overlooked it)

Reply:
Ok, just saw your last commit - I'll shut up now and let you work ;)

Reply:
Only saw it because of your comment --  keep them coming,..